scripts/fetch_api.sh

```diff
@@ -2,12 +2,17 @@
 
 # get the download URL of the OpenAPI spec file
 OPENAPI_FILE_URL=$1
-OPENAPI_FILE_NAME=$(dirname $0)/../.openapi/$(basename $OPENAPI_FILE_URL)
+OPENAPI_FILE_NAME=$(basename $OPENAPI_FILE_URL)
+
+# Special case when downloading Service Registry Instance schema file,
+# to keep the original (manually updated) file name.
+if [ $OPENAPI_FILE_NAME == "openapi.json" ]; then
```

Comment:
This is not very elegant, an may cause issues if any other schema is named `openapi.json`, but it's the most straightforward solution. I'm open to suggestions if this needs to be changed.

Reply:
can we rename the openapi.json to something more specific? registry-instance.json or something?

Reply:
I'd rather it's handled on the consuming (this) side.

Reply:
I'll discuss with the team. This API comes from the upstream dataplane however, and is not strictly a MAS API.

Reply:
Moreover, we would have to rename it to `registry-instance.json` to avoid changing anything here, so if we decide to rename it to something admittedly better like `apicurio_registry_v2.json`, the issue is still present.

Reply:
like I said, we can handle it on this side if it is very painful for your team but I'd like to see the conventions being stuck to. This isn't a big deal either way really 😄.